### DISCUSSION TITLE
I need help

### DISCUSSION THREAD

**Discussion by Hprice00** [community/community#141831] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

So I have a project im currently working on, and when i add files to my .gitignore, it never actually ignores the files. it shows up that i added them but never does, so i end up always having to discard all of my changes everytime i restart my intellij. can anyone help out?

**Comment by Berchez** [comment]
If files are still tracked despite being in .gitignore, follow these steps:

1. Check .gitignore: Ensure file paths are correct.

2. Untrack Files: Run:
`git rm --cached <file-path>`

3.Commit:
`git commit -m "Remove tracked files and update .gitignore"`

4. Push:
`git push`

This will stop Git from tracking the files listed in .gitignore.

**Reply by Hprice00** [reply]
Where do i input the "git commit -m "Remove tracked files and update .gitignore" and the "git push" because i try in the terminal and nothing happens.

**Reply by Berchez** [reply]
Where you have your git installed.
See [this tutorial](https://www.simplilearn.com/tutorials/git-tutorial/git-installation-on-windows)
